assignment5.js

```diff
@@ -0,0 +1,30 @@
+var fs = require('fs');
+
+var fileReader = (fileName) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile(fileName, 'utf8', (error, data) => {
+      console.log(data); 
+      console.log("============");
+
+      if(error)
+        return reject(error);
+      resolve(fileName + 'fileDone');
```

Comment:
Are you using the string sent by resolve. If not, give a valid reason of having this string here.